Sources/_TestingInternals/include/Includes.h

```diff
@@ -27,7 +27,7 @@
 /// - Note: Avoid including headers that aren't actually used.
 
 #include <errno.h>
-#if __has_include(<signal.h>)
+#if __has_include(<signal.h>) && !defined(__wasi__)
```

Comment:
What goes wrong if signal.h is included on WASI? Is it harmful?

Reply:
I see you commented about this in the commit description. Please document the issue in a comment here so we can refer back to it in the future. Thanks!

Reply:
Added a comment there :)